tests/layer/test_deep_graph_infomax.py

```diff
@@ -43,14 +47,37 @@ def test_dgi():
     assert embeddings.shape == (len(G.nodes()), emb_dim)
 
 
-def test_dgi_embedding_model_wrong_model():
+@pytest.mark.parametrize("model_type", [GCN, APPNP, GAT])
```

Comment:
No PPNP here because it doesn't support sparsity?

Reply:
yep that's right.